code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -112,6 +112,32 @@ namespace WijmoProvider.Feature {
             return false;
         }
 
+        /**
+         * Checks if a specific row is invalid by checking if it contains any invalid cells.
+         * @param rowKey Key of the row to check if a row has invalid cells
+         * @returns Boolean that indicates if the row is invalid. Returns True if invalid. False, otherwise.
+         */
+        private _isInvalidRowByRowKey(rowKey: string): boolean {
```

Comment:
can we optimize this method as is looks a lot similar to the older "_isInvalidRowByRowNumber"

Reply:
Please check now.